### ISSUE TITLE
contact.onIncoming - accept does not work unless the ccp-v2 is open

### ISSUE THREAD

**Issue by zachryzion** [amazon-connect/amazon-connect-streams#263]
I am accepting calls on incoming contacts. However, for the first call is not accepted unless I have the ccp-v2 is open. I am not sure why is that. Any help will be appreciated.

`contact.onConnecting(handleContactConnecting);`
```  function handleContactConnecting(contact: connect.Contact) {
    console.log("incoming contact", contact, contact.getAttributes());
    console.log("[contact.onIncoming] Contact is incoming. Contact state is " + contact.getStatus().type);
    if (available){
      contact.accept({
        success: function() {
          console.log("handleContactConnecting - available - Incoming contact accepted via streams");
        },
        failure: function() {
          console.log("handleContactConnecting - available - [ERROR] Failed to accept incoming contact via Streams");
        },
      });
    }
  }```

**Comment by ctwomblyamzn** [comment]
Hi @zachryzion, I don't know if there is quite enough information here, but let me ask you this question:

When you say "the first call is not accepted unless I have the ccp-v2 open" do you mean: 
     a. The first call is not accepted unless I open ccp-v2 in a new tab
     If this is the case, which I suspect may be the case, you need to be aware that the `softphone.allowFramedSoftphone` parameter for `initCCP` needs to be set to `true` if you want your embedded CCP to take care of voice media communications. See the docs for `initCCP` [here](https://github.com/amazon-connect/amazon-connect-streams/blob/master/Documentation.md#connectcoreinitccp).
     b. The first call is not accepted unless I specify `ccpUrl` in `initCCP` as ".../ccp-v2" instead of "../ccp"
     If this is the case, then I would definitely like to see your `initCCP` parameters, to make sure there is nothing awry. Using the old CCP should work perfectly even with the newest streams releases. 

If neither of these options cover your implementation, then please give me more details so that I can assist you further :)

**Comment by zachryzion** [comment]
Hi @ctwomblyamzn,
_a. The first call is not accepted unless I open ccp-v2 in a new tab_
This was the case. I did not try `softphone.allowFramedSoftphone` as I was specifically using the chat feature.

_b. The first call is not accepted unless I specify ccpUrl in initCCP as ".../ccp-v2" instead of "../ccp"_
This worked. I used "../ccp#" and that was not accepting the calls, unless I used "../ccp-v2"
